### ISSUE TITLE
codesign.dart needs a way to be manually invoked by release engineers

### ISSUE THREAD

**Issue by CaseyHillers** [flutter/flutter#132678]
The last AI before we can deprecate codesign.py is to ensure release engineers have a back up way of codesigning binaries (in case automation fails, or there is an outage).

There's a few ways this could be done, but I presume creating an MPA script that schedules codesigning drone builds for a given hash would be sufficient, and allow us to reuse our existing infra.

**Comment by CaseyHillers** [comment]
Friendly ping. This became a blocker on the 3.16 stable release as we're still relying on codesign.py to ship releases.

**Comment by XilaiZhang** [comment]
from triage: should have a way to run codesign.dart separate from the main build process. potentially still on dart internal and through MPA

**Comment by matanlurey** [comment]
Closing as stale. Please feel free to open a new issue with a strong justification for doing this.

(To be clear, the `-release` and `-infra` teams have no plans to work on this in the known future)
